projects/igniteui-angular/src/lib/grids/grid/grid-mrl-keyboard-nav.spec.ts

```diff
@@ -1,2903 +1,3104 @@
-        fix.componentInstance.colGroups = [{
-            group: 'group1',
-            hidden: true,
-            columns: [
-                { field: 'ID', rowStart: 1, colStart: 1, rowEnd: 3 }
-            ]
-        }, {
-            group: 'group2',
-            columns: [
-                { field: 'CompanyName', rowStart: 1, colStart: 1, colEnd: 3 },
-                { field: 'ContactName', rowStart: 2, colStart: 1 },
-                { field: 'ContactTitle', rowStart: 2, colStart: 2 },
-                { field: 'Address', rowStart: 3, colStart: 1, colEnd: 3 }
-            ]
-        }, {
-            group: 'group3',
-            columns: [
-                { field: 'City', rowStart: 1, colStart: 1, colEnd: 3, rowEnd: 3 },
-                { field: 'Region', rowStart: 3, colStart: 1 },
-                { field: 'PostalCode', rowStart: 3, colStart: 2 }
-            ]
-        }, {
-            group: 'group4',
-            columns: [
-                { field: 'Country', rowStart: 1, colStart: 1 },
-                { field: 'Phone', rowStart: 1, colStart: 2 },
-                { field: 'Fax', rowStart: 2, colStart: 1, colEnd: 3, rowEnd: 4 }
-            ]
-        }];
-        fix.detectChanges();
-        const firstCell = fix.debugElement.queryAll(By.css(CELL_CSS_CLASS))[0];
+            it('should navigate using Arrow Right through bigger cell with smaller rowStart and bigger rowEnd', () => {
+                fix.componentInstance.colGroups = [{
+                    group: 'group1',
+                    columns: [
+                        { field: 'ID', rowStart: 1, colStart: 1 },
+                        { field: 'CompanyName', rowStart: 1, colStart: 2, rowEnd: 3 },
+                        { field: 'ContactName', rowStart: 1, colStart: 3 },
+                        { field: 'ContactTitle', rowStart: 2, colStart: 1 },
+                        { field: 'City', rowStart: 2, colStart: 3 },
+                        { field: 'Country', rowStart: 3, colStart: 1 },
+                        { field: 'Address', rowStart: 3, colStart: 2, colEnd: 4 },
+                    ]
+                }];
+                fix.detectChanges();
 
-        UIInteractions.clickElement(firstCell);
-        await wait();
-        fix.detectChanges();
+                let dummyCell;
+                let firstCell;
+                let secondCell;
+                let thirdCell;
+                [
+                    dummyCell, secondCell, thirdCell,
+                    firstCell,              dummyCell
+                ] = fix.debugElement.queryAll(By.css(CELL_CSS_CLASS));
 
-        UIInteractions.triggerKeyDownWithBlur('arrowright', firstCell.nativeElement, true, false, false, true);
-        await wait(DEBOUNCETIME);
-        fix.detectChanges();
+                firstCell.triggerEventHandler('focus', null);
+                fix.detectChanges();
 
-        expect(fix.componentInstance.selectedCell
```

Comment:
Can you add this describe under describe('IgxGrid Multi Row Layout - Keyboard navigation #grid') or add tag grid for it